geopandas/io/tests/test_file.py

```diff
@@ -310,6 +312,11 @@ def test_to_file_column_len(tmpdir, df_points):
 @pytest.mark.parametrize("driver,ext", driver_ext_pairs)
 def test_append_file(tmpdir, df_nybb, df_null, driver, ext):
     """Test to_file with append mode and from_file"""
+
+    if ext == ".tab":
+        # should there be a warning when this happens?
+        pytest.xfail("MapInfo File doesn't maintain accuracy.")
```

Comment:
What does it mean exactly?

Reply:
with the `xfail` that we use mentioned below, upon read-in after a write-out of the following gdf in the `test_empty_crs`

```
    df = GeoDataFrame({"a": [1, 2, 3], "geometry": [Point(0, 0), Point(1, 1), Point(2, 2)]})
```
we see the lack of precision after the read-in coming `assert_geodataframe_equal` and is outside the bounds of using `check_less_precise` flag, i.e. it still fails with the flag on.

```
left = 0    POINT (0.00000 0.00000)
1    POINT (0.99000 1.00000)
2    POINT (2.01000 2.00000)
Name: geometry, dtype: geometry
right = 0    POINT (0.00000 0.00000)
1    POINT (1.00000 1.00000)
2    POINT (2.00000 2.00000)
Name: geometry, dtype: geometry

```

This is in addition to the fact that the MapInfo driver writes out the default CRS that I mention